bot.py

```diff
@@ -128,6 +128,13 @@ def process_message_inner(config, message, conv):
         conv.set_debug(False)
         return keybase.send(conv.id, NODEBUG)
 
+    elif msg_type == parse.MSG_SNOOZE:
+        if conv.context != conversation.CTX_REMINDED:
```

Comment:
The pattern I had been using was to have parse only return the result if it makes sense for the context. So in this case it would only be possible to parse snooze in CTX_REMINDED. Want to do the same thing here? On the one hand it makes the generic MSG_UNKNOWN happen more often but on the other hand we lower potential mis-parses and also we always log on MSG_UNKNOWN so we can spot and fix if it's an issue.

Reply:
Parse now only parses snooze if `CTX_REMINDED`. I still having this check so the program doesn't if parse changes in the future.